docs/api/navigators/StackNavigator.md

```diff
@@ -108,6 +108,10 @@ String or React Element used by the header. Defaults to scene `title`
 
 Title string used by the back button on iOS, or `null` to disable label. Defaults to the previous scene's `headerTitle`
 
+#### `headerBackImage`
+
+Image asset used by the back button.
```

Comment:
It would be helpful to add more documentation about the type of this prop. Does the user set `headerBackImage` by using `require` like `headerBackImage={require('path/to/image')}` or can it be dynamic with `headerBackImage={{uri: 'path/to/image'}}`? These are questions that should be answered by the documentation.